admin/class-h5p-content-admin.php

```diff
@@ -206,6 +224,13 @@ public function display_contents_page() {
         return;
 
       case 'show':
+        // Access restriction
+        if ($this->current_user_can_view($this->content) == FALSE) {
+          H5P_Plugin_Admin::set_error(__('You are not allowed to view this content.', $this->plugin_slug));
+          H5P_Plugin_Admin::print_messages();
+          return;
+        }
```

Comment:
Any reason we choose to not use the default access denied from WP?

Reply:
I chose to stay consistent with how the access restrictions for editing content is implemented: https://github.com/h5p/h5p-wordpress-plugin/blob/2b339d60bb6a1d03ddc757dfd18a9b322246dc5d/admin/class-h5p-content-admin.php#L317-L319

Reply:
@icc  Just let me know if those two should be changed, please.

Reply:
Then it's OK. I will merge everything together and do some more testing.

Reply:
Can you resolve the conflicts first?

Reply:
Will do ...

Reply:
@icc This should do.